lua/nvim-treesitter/parsers.lua

```diff
@@ -704,14 +704,39 @@ list.gleam = {
 
 list.glimmer = {
   install_info = {
-    url = "https://github.com/alexlafroscia/tree-sitter-glimmer",
+    url = "https://github.com/ember-tooling/tree-sitter-glimmer",
+    branch = "main",
     files = { "src/parser.c", "src/scanner.c" },
   },
   filetype = "handlebars",
   maintainers = { "@NullVoxPopuli" },
   readme_name = "Glimmer and Ember",
 }
 
+list.glimmer_javascript = {
+  install_info = {
+    url = "https://github.com/NullVoxPopuli/tree-sitter-glimmer-javascript",
+    branch = "main",
+    files = { "src/parser.c", "src/scanner.c" },
+    generate_requires_npm = true,
+  },
+  filetype = "javascript.glimmer",
+  maintainers = { "@NullVoxPopuli" },
+  readme_name = "Glimmer-flavored JavaScript",
+}
+
+list.glimmer_typescript = {
+  install_info = {
+    url = "https://github.com/NullVoxPopuli/tree-sitter-glimmer-typescript",
+    branch = "main",
+    files = { "src/parser.c", "src/scanner.c", "src/tree-sitter-typescript/scanner.h" },
```

Comment:
It is, it's just two fires in the same repo instead of one

Reply:
ah wait! I think I get it -- that's why tree-sitter-typescript has a makefile here: https://github.com/tree-sitter/tree-sitter-typescript/blob/master/typescript/Makefile

the compile their stuff in to one file, rather than pass it downstream.
I will fix

Reply:
Makefiles are also not acceptable. Just copy the header.

Reply:
I mean, I did copy the header -- here: https://github.com/NullVoxPopuli/tree-sitter-glimmer-typescript/blob/main/src/tree-sitter-typescript/scanner.h :sweat_smile: 

Why is a Makefile not acceptable? how would a consumer even know, if the output ultimately ends up being what defaults are?

Reply:
Because we rely on a simple, sane build step. We simply cannot support 300 snowflake parsers. Copy the scanner **including its header** into each "dialect" subdirectory; this is non-negotiable and I don't want to waste time arguing about it.

(And, no, symlinks are not an option; these don't work on Windows.)

Reply:
Gotchya -- I appreciate you're patience with me on this -- I've had a lot of loosey goosey misconceptions in my head to work through.

.h is removed now.